src/centralstore/incremental/NodeBlock.h

```diff
@@ -0,0 +1,56 @@
+#include <cstring>
+#include <fstream>
+#include <iostream>  // for cout
+#include <set>
+#include <string>
+
+#include "PropertyLink.h"
+
+#ifndef NODE_BLOCK
+#define NODE_BLOCK
+
+class NodeBlock {
+   private:
+    unsigned int addr;
+
+   public:
+    std::string id;  // Node ID for this block ie: citation paper ID, Facebook accout ID, Twitter account ID etc
+    char usage;      // Wheather this block is in use or not
```

Comment:
Typo 'Wheather'

Reply:
Thanks,
Fixed in https://github.com/miyurud/jasminegraph/pull/104/commits/9f35b95f13fccc96e0b3963e753b5efde0c6b89f

Reply:
Done